### PR TITLE
[ADD] udes_stock: Add indexes for stock.move.line

### PR DESCRIPTION
**Pull request by squah-u**
Task: 9993
User-story: 9881

Signed-off-by: Sean Quah <sean.quah@unipart.io>

### CODE REVIEW

File: addons/udes_stock/models/stock_move_line.py [hunk 1]
```diff
@@ -1038,6 +1038,25 @@ def _prevent_write_after_done(self):
         if done_lines:
             raise ValidationError(_("Cannot update move lines that are already 'done'."))
 
+    @api.model_cr
+    def init(self):
+        """ Creates indexes for _check_resultant_package_level
+        """
+        super(StockMoveLine, self).init()
+
+        tools.create_index(
```
Comment (kdwyer) [inline addons/udes_stock/models/stock_move_line.py:1047]: :+1: I didn't know about these tools.

File: addons/udes_stock/models/stock_move_line.py [hunk 2]
```diff
@@ -1038,6 +1038,25 @@ def _prevent_write_after_done(self):
...
+            self._cr,
+            "stock_move_line_state_result_package_id_index",
+            self._table,
+            ["state", "result_package_id"],
```
Comment (kdwyer) [inline addons/udes_stock/models/stock_move_line.py:1051]: If we're theoretically targeting Python3.5, I think the trialing comma on the final arguemnt isn't valid syntax.

File: addons/udes_stock/models/stock_move_line.py (hunk 1 above)
Comment (squah-u) [reply addons/udes_stock/models/stock_move_line.py:1047]: It's very much copied from Miquel's PR

File: addons/udes_stock/models/stock_move_line.py (hunk 2 above)
Comment (squah-u) [reply addons/udes_stock/models/stock_move_line.py:1051]: Here I was following the existing style for this file (eg. line 822)

**Pull request merged**